docs/source/markdown/podman-untag.1.md

```diff
@@ -0,0 +1,37 @@
+% podman-untag(1)
+
+## NAME
+podman\-untag - Removes one or more names from a locally-stored image
+
+## SYNOPSIS
+**podman untag** *image*[:*tag*] *target-names*[:*tag*] [*options*]
+
+**podman image untag** *image*[:*tag*] *target-names*[:*tag*] [*options*]
```

Comment:
The usage here and in the commands help message implies that specifying at least on name is necessary but they are optional. Putting it in brackets would fix it.

Reply:
Fixed :+1: